main.rb

```diff
@@ -0,0 +1,130 @@
+require './music_album'
+require './genre'
+require './book'
+require './game'
+require './item_data'
+
+class Main
+  def display_menu
+    puts 'Please choose an option to enter in a number'
+    puts '1. List all books'
+    puts '2. List all music albums'
+    puts '3. List of games'
+    puts '4. List all genres (e.g. "Hip Hop", "Rock")'
+    puts '5. List all labels (e.g. "Gift", "New")'
+    puts '6. List all authors (e.g. "Stephen King", "J.K. Rowling")'
+    puts '7. List all sources (e.g. "From a friend", "Amazon")'
+    puts '8. Add a book'
+    puts '9. Add a music album'
+    puts '10. Add a game'
+    puts '11. Exit'
+  end
+
+  def load_book
+    puts ItemData.load('book')
+  end
+
+  def load_music_album
+    puts ItemData.load('music_album')
+  end
+
+  def load_game
+    puts ItemData.load('game')
+  end
+
+  def list_genres
+    MusicAlbum.list_genres
+  end
+
+  def list_labels
+    Item.list_labels
+  end
+
+  def list_sources
+    Item.list_sources
+  end
+
+  def list_authors
+    Book.list_authors
+  end
+
+  def add_book_item
+    ItemData.add_item('book', Book.create_from_ui)
+    puts 'Book added'
+  end
+
+  def add_music_album_item
+    ItemData.add_item('music_album', MusicAlbum.create_from_ui)
+    puts 'Music album added'
+  end
+
+  def add_game_item
+    ItemData.add_item('game', Game.create_from_ui)
+    puts 'Game added'
+  end
```

Comment:
- [ ] I am unable to add a game, please checkout the screenshot below and kindly fix the issue. 

![add_game_error](https://user-images.githubusercontent.com/65852621/229307312-f1d1b37e-6bca-46e0-8bba-4b58addd0fd3.png)